### ISSUE TITLE
Sinatra Version

### ISSUE THREAD

**Issue by John-Nenniger** [jruizgit/rules#171]
does it need to be pinned to version 1.4?

**Comment by jruizgit** [comment]
No, there is nothing specific to version 1.4. Let me test the latest version and I will remove the `.4` requirement.
